pom.xml

```diff
@@ -119,6 +120,18 @@
       <version>2.2</version>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>com.github.tomakehurst</groupId>
```

Comment:
Why do we need this dependency?

Reply:
I've removed it, thanks, it's no longer valid dependency